src/test/java/HistoryTest.java

```diff
@@ -0,0 +1,110 @@
+import org.junit.jupiter.api.Test;
+
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HistoryTest {
+
+    /**
+     * testListHistory is a JUnit test of the method listHistory in the history class. The function is creating
+     * three files in a test_folder, if listHistory returns the name of those files with timestamps in ascending orders the
+     * test case is considered successful. All other outputs are considered as a fail.
+     * <p>
+     * Input: create files test_build_history/test 20T03T20T30 test_build_history/test 20T01T20T30 test_build_history/test 20T03T19T30
+     * Expected output: test 20T01T20T30 test 20T03T19T30 test 20T03T20T30
+     */
+    @Test
+    public void testListHistory() {
+        History history = new History(true);
+
+        File file = new File("test_build_history/test 20T03T20T30");
+        File file1 = new File("test_build_history/test 20T01T20T30");
+        File file2 = new File("test_build_history/test 20T03T19T30");
+        try {
+
+            FileWriter writer = new FileWriter(file);
+            writer.write("hello");
+            writer.close();
+
+            writer = new FileWriter(file1);
+            writer.write("hello");
+            writer.close();
+
+            writer = new FileWriter(file2);
+            writer.write("hello");
+            writer.close();
+
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        try {
+            String correct = "test 20T01T20T30\ntest 20T03T19T30\ntest 20T03T20T30\n";
+            assertTrue(history.listHistory().compareTo(correct) == 0);
```

Comment:
isn't is better to write this line as
`assertEquals(correct, history.listHistory()) == 0);`
instead, because `correct` is the expected value and `history.listHistory()) == 0` is the actual value?